tests/suite/errors/nosemi.yaml

```diff
@@ -0,0 +1,12 @@
+script: zq in.tzng
```

Comment:
nit: it looks like this test can be written without script and with errorRE like this one: https://github.com/brimsec/zq/blob/master/tests/suite/put/put-6.yaml which would be a little more concise

Reply:
thanks, i'll do that, i'm not keeping up with our test capabilities...